Tests/ReactiveSwiftTests/SignalProducerSpec.swift

```diff
@@ -793,6 +793,14 @@ class SignalProducerSpec: QuickSpec {
 				expect(dates) == [tick1, tick2, tick3]
 			}
 
+			it("shouldn't overflow on a real scheduler") {
+				let scheduler = QueueScheduler()
+				let producer = timer(interval: 3, on: scheduler)
```

Comment:
If the crash happened with `> 9` seconds, why not use `10` here to make sure this catches a potential regression? :)